### PR TITLE
Document how to run in-tree cloud providers with kubeadm

### PR DESCRIPTION
**Pull request by dims**
We need some documentation on how to cleanly specify the information about standing up in-tree cloud providers with kubeadm with the latest configuration info from v1.12

Change-Id: Iab6b996a830503d74a6eb0c507c5f8ca7a39235b

### CODE REVIEW

**Comment by dims** [comment]
cc @detiber @neolit123 @timothysc

**Comment by neolit123** [comment]
LGTM on my side until 1.13 is here.

@kubernetes/sig-cluster-lifecycle-pr-reviews 
/assign @timothysc

File: content/en/docs/concepts/cluster-administration/cloud-providers.md [hunk 1]
```diff
@@ -9,7 +9,47 @@ This page explains how to manage Kubernetes running on a specific
...
+apiVersion: kubeadm.k8s.io/v1alpha3
+kind: InitConfiguration
+nodeRegistration:
+  kubeletExtraArgs:
+    cloud-provider: "openstack"
+    cloud-config: "/etc/kubernetes/cloud.conf"
+---
+kind: ClusterConfiguration
+apiVersion: kubeadm.k8s.io/v1alpha3
+kubernetesVersion: v1.12.0-rc.1
```
Comment (neolit123) [inline content/en/docs/concepts/cluster-administration/cloud-providers.md]: we might want to remove the `-rc.1`?

Comment (dims) [reply content/en/docs/concepts/cluster-administration/cloud-providers.md]: oops. fixed.

**Comment by neolit123** [comment]
thanks
/lgtm

**Pull request merged**